clients/da-vinci-client/src/main/java/com/linkedin/davinci/blob/BlobResponse.java

```diff
@@ -0,0 +1,51 @@
+package com.linkedin.davinci.blob;
+
+import java.util.List;
+import java.util.Objects;
+
+
+public class BlobResponse {
+  private boolean isError;
+
+  private String message;
+
+  private List<String> partitionURLs;
```

Comment:
What are `partitionURLs`? Are they the urls of the host?

Reply:
Made some of those changes in a previous PR